src/main/kotlin/uk/gov/justice/digital/hmpps/notificationsalertsvsip/service/NotificationService.kt

```diff
@@ -26,7 +26,7 @@ class NotificationService(
 ) {
   private companion object {
     val LOG: Logger = LoggerFactory.getLogger(this::class.java)
-    private const val SMS_DATE_PATTERN = "dd MMMM yyyy"
+    private const val SMS_DATE_PATTERN = "d MMMM yyyy"
```

Comment:
updated date pattern from dd to d. This will format date to single date if date is before the 10th - https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html